packages/rs-drive-abci/.env.testnet

```diff
@@ -23,11 +23,12 @@ CORE_JSON_RPC_PASSWORD=password
 
 INITIAL_CORE_CHAINLOCKED_HEIGHT=1243
 
-# https://github.com/dashevo/dashcore-lib/blob/286c33a9d29d33f05d874c47a9b33764a0be0cf1/lib/constants/index.js#L42-L57
-VALIDATOR_SET_LLMQ_TYPE=llmq_25_67
+# https://github.com/dashpay/dashcore-lib/blob/5a3ddfc3588019b90f436766aeff4ffdc5d8f41b/lib/constants/index.js#L47-L96
```

Comment:
I don't know if this link is actually relevant. I don't see a 1:1 mapping between the two.